DESCRIPTION

```diff
@@ -13,12 +13,16 @@ RoxygenNote: 7.3.2
 Imports:
     hubUtils,
     dplyr,
-    hubExamples
+    hubExamples,
+    hubEvals,
+    hubEnsembles
```

Comment:
maybe worth confirming whether you are relying on a minimum specific version of these packages.